carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/core/factory/impl/DesktopFactory.java

```diff
@@ -180,12 +180,13 @@ public Boolean apply(WebDriver driver ) {
                         driver.manage().window().setSize(new Dimension(width, height));
                         if (driver.manage().window().getSize().getWidth() == width
```

Comment:
try to minimize number of driver calls as well, i.e. do a single call 
size = driver.manage().window().getSize()

and opearet later with this object